pkg/pillar/cmd/nodeagent/handletimers.go

```diff
@@ -101,6 +102,25 @@ func handleUpgradeTestValidation(ctxPtr *nodeagentContext) {
 	}
 }
 
+// when baseos upgrade is inprogress
+// check if vault is accessible, and if not reset the node
+func handleFallbackOnVaultLocked(ctxPtr *nodeagentContext) {
+	if !ctxPtr.updateInprogress || ctxPtr.vaultOperational {
```

Comment:
FWIW the SIGUSR I did to sc-supermicro-zc1 was just after a reboot and not an update, so perhaps we should also cause a reboot in the non-update case??
Thoughts?

Reply:
Addressed.